packages/client/components/MassInvitationTokenLink.tsx

```diff
@@ -77,13 +78,22 @@ const MassInvitationTokenLink = (props: Props) => {
   const displayToken = isTokenValid ? token : '············'
   const linkLabel = `${window.__ACTION__.prblIn}/${displayToken}`
   const url = getMassInvitationUrl(displayToken)
+
+  const {viewerId} = atmosphere
+  const handleCopy = () => {
+    SendClientSegmentEventMutation(atmosphere, 'Copied Team Invite Link', {
```

Comment:
We should follow [Segment naming convention](https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/) (Objection-action framework), so let's rename the event name to "Team Invitation Link Copied"

Reply:
Good to know, thanks @tianrunhe , I just already updated.